learnrl/playground.py

```diff
@@ -356,19 +359,30 @@ def run(self,
                     callbacks.on_steps_cycle_end(step, logs)
 
                 step += 1
-
+                logs['cur_step'] += 1
+                if steps >= 0 and logs['cur_step'] >= steps:
```

Comment:
For-break can lead to unwanted behavior if the number of episodes is too small.
It would be preferable to have a while loop that encapsulate both episodes or steps defined runs.